samples/boards/stm32/power_mgmt/blinky/README.rst

```diff
@@ -16,6 +16,11 @@ For example, when clocking the  low power Timer with LSE clock at 32768Hz and ad
 prescaler of <32>, then the kernel sleep period can reach 65536 * 32/32768 = 64 seconds
 CONFIG_SYS_CLOCK_TICKS_PER_SEC is set to 1024.
 
+To sleep forever, call the k_msleep()
+ - with **INT_MAX** when CONFIG_SYSTEM_CLOCK_SLOPPY_IDLE is not set
+ - with **K_TICKS_FOREVER** when CONFIG_SYSTEM_CLOCK_SLOPPY_IDLE is set
```

Comment:
This isn't related to blinky PM. We need to find another way to document this if needed.

Reply:
not really, that was just a example.
Removed